versioned_docs/version-1.3-preview/commands/oras_attach.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: oras attach
+sidebar_position: 10
+warning: Do NOT modify this generated file
+---
+
+# oras attach
+
+[Preview] Attach files to an existing artifact
+** This command is in preview and under development. **
```

Comment:
Did we re-generate the command help docs based on the latest binary? `oras attach` is not in preview anymore, and some flags seems missing. Similar issues for other commands.

Reply:
Do we have a guide on how to use the script?